src/main/kotlin/org/rust/ide/core/overrideImplement/ImplementMembersHandler.kt

```diff
@@ -0,0 +1,27 @@
+package org.rust.ide.core.overrideImplement
+
+import com.intellij.lang.LanguageCodeInsightActionHandler
+import com.intellij.openapi.editor.Editor
+import com.intellij.openapi.project.Project
+import com.intellij.psi.PsiFile
+import org.rust.lang.core.psi.RsImplItem
+import org.rust.lang.core.psi.impl.RsFile
+import org.rust.lang.core.psi.util.parentOfType
+
+class ImplementMembersHandler: LanguageCodeInsightActionHandler {
+    override fun isValidFor(editor: Editor, file: PsiFile): Boolean {
+        if (file !is RsFile)
+            return false
+        val elementAtCaret = file.findElementAt(editor.caretModel.offset)
+        val classOrObject = elementAtCaret?.parentOfType<RsImplItem>(strict = false)
+        return classOrObject != null 
+    }
+
+    override fun startInWriteAction() = false
+
+    override fun invoke(project: Project, editor: Editor, file: PsiFile) {
+        val elementAtCaret = file.findElementAt(editor.caretModel.offset)
+        val implItem = elementAtCaret?.parentOfType<RsImplItem>(strict = false) ?: error("No impl trait item")
```

Comment:
I think we should not `error` here, and just do nothing? 

`error` is useful to check invariants about PSI you control. For example, if you use `psiFactory` to generate a function, you can be sure that it has a body, and you want to assert this in code. 

In contrast, when the PSI is supplied by the user, it can be broken in different ways, and one should handle `null` gracefully, either by doing nothing, or by showing `Action is not applicable` error popup.